api/share/requests/users.py

```diff
@@ -0,0 +1,65 @@
+import re
+from api.share.requests import ValidRequestObject, InvalidRequestObject
+
+
+class GetUsersCountRequest(ValidRequestObject):
+
+    def __init__(self,
+                 registration_bot_chat_id: str,
```

Comment:
this doesn't look like it conforms with `black`...

Reply:
Yes, I know